app/controllers/api/v1/topics_controller.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module V1
     class TopicsController < Api::V1::ApiController
       def index
-        @topics = policy_scope(Topic)
+        @topics = policy_scope(Topic).include([:icon_url])
```

Comment:
minor code enhancement:
```suggestion
        @topics = policy_scope(Topic).include(:icon_url)
```